src/app/ws-upload/upload-model.ts

```diff
@@ -11,6 +11,7 @@ export class UploadModel {
     public file: File;
   
     constructor() {
-      this.id = UUID.UUID();
+      this.id = UUID.UUID();//suan icin ne oldugunu bilmiyoruz
```

Comment:
Try to resist comments being committed in any language other than English, otherwise Alexey will be encouraged and then we'll have Russian everywhere too :)